nixos/modules/services/web-servers/apache-httpd/default.nix

```diff
@@ -613,7 +628,13 @@ in
       enableCompression = mkOption {
         type = types.bool;
         default = false;
-        description = "Enable compression of responses using mod_deflate.";
+        description = "Enable compression of responses using <literal>mod_deflate</literal>.";
+      };
+
+      verifyConfig = mkOption {
```

Comment:
I was reasoning that there might be edge cases where the test fails during build and wanted to provide an out.